.github/workflows/ci.yml

```diff
@@ -14,25 +15,30 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Use Node.js 16.x
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
           node-version: 16.x
       - run: yarn --immutable --ignore-engines
-
-      - name: "Check Code Format"
-        run: yarn prettier:check
-
-      - name: "Lint Code"
-        run: yarn lint
```

Comment:
I don't see the formatting and linting rules in the new CI file; am I missing something?

Reply:
Ooops. You're right, I added them to the build step.